src/patternfly/components/Select/select.scss

```diff
@@ -443,6 +451,23 @@
   padding-top: var(--pf-c-select__menu-group--not-first--PaddingTop);
 }
 
+.pf-c-select__menu-group {
+  &.pf-m-padding {
```

Comment:
Why use a modifier here for padding? Wouldn't it always need the padding? If it is a modifier should it be called padding? I didn't think we named them like this.

Reply:
Yeah, we're using `.pf-m-padding` in some components (`.pf-c-drawer` most recently) where padding is a boolean and pre-configured. This is what I thought to be the most flexible approach as we build out functionality in the future opting for existing, generic elements over specific elements like `.pf-c-select__filter`.  Wdyt?

Reply:
ah gotcha so more reusable this way. Ok, I'm behind on the newest rules for PF :)

Reply:
and that `var` is wrong, lol